src/application.ts

```diff
@@ -7,6 +7,9 @@ import { GitHubAPI } from './github'
 import { logger } from './logger'
 import { LoggerWithTarget, wrapLogger } from './wrap-logger'
 
+// Cache for 1 minute less than GitHub expiry
+const installationTokenTTL = parseInt(process.env.INSTALLATION_TOKEN_TTL || '3540', 10)
```

Comment:
I don't think the use of `parseInt` is clear or necessary here. Why parse an int into base 10 when we can just require users to use the env variable that way?

Reply:
`process.env` always contains strings, so casting to other types is necessary before use. `parseInt` is used to cast a string to a number - the default is base `10`, but `tslint` complains if it's not set explicitly.

See https://repl.it/repls/AdvancedElatedMapping

Reply:
👍 gotcha! I was confused at use of `parseInt()` instead of `Number()`, but doing some googling tells me your solution is probably better!